### PR TITLE
Our commands should only be available if we're on a Wynncraft server

### PR DESCRIPTION
**Pull request by magicus**
Thanks to @coehlrich for noticing this.

### CODE REVIEW

**Review by kristofbolyai — DISMISSED** [review]

**Review by HighCrit — CHANGES_REQUESTED** [review]
No translation lines?

File: common/src/main/java/com/wynntils/core/commands/ClientCommandManager.java [hunk 1]
```diff
@@ -59,16 +58,15 @@ private static void registerCommand(CommandBase command) {
     }
 
     @SubscribeEvent
```
Comment (coehlrich) [inline common/src/main/java/com/wynntils/core/commands/ClientCommandManager.java]: I think you forgot to remove this line

File: common/src/main/java/com/wynntils/features/user/CommandsFeature.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.commands.ClientCommandManager;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.mc.event.ChatSentEvent;
+import com.wynntils.wc.utils.WynnUtils;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class CommandsFeature extends UserFeature {
```
Comment (HighCrit) [inline common/src/main/java/com/wynntils/features/user/CommandsFeature.java:13]: Also do we want this feature to be enabled/disabled by the user?

File: common/src/main/java/com/wynntils/core/commands/ClientCommandManager.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/core/commands/ClientCommandManager.java]: Yep. Nice spotted!

File: common/src/main/java/com/wynntils/features/user/CommandsFeature.java (hunk 2 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/features/user/CommandsFeature.java:13]: Yes, I think it makes sense. Once again, if we do something bad. Or if, let's say Wynncraft introduces a `/feature` command, the user can disable this. Most of the features that the user can disable makes no real sense (in the end, if you disable all of them you might as well not use Wynntils :)) but I think it makes sense that each and every bit of modification we do can be individually turned off.

**Comment by magicus** [comment]
> No translation lines?

I keep forgetting those. :( '/feature list' shows I'm not the only one. ;-) We should have a check for this in FeatureRegistry, really.

**Comment by kristofbolyai** [comment]
> > No translation lines?
> 
> 
> 
> I keep forgetting those. :( '/feature list' shows I'm not the only one. ;-) We should have a check for this in FeatureRegistry, really.

I would rather do this in spotless with a custom module

**Comment by magicus** [comment]
> I would rather do this in spotless with a custom module

Sounds like the proper place, yeah. But also much more work. What did you call it? "Sounds nice and really complicated."? :-D If you want to do it (after your vacation, no hurry) you're more than welcome. Also, ideally we'd also check functions, config variables and overlays.

**Review by HighCrit — APPROVED** [review]

**Pull request merged**